gitlab/resource_gitlab_service_pipelines_email.go

```diff
@@ -0,0 +1,114 @@
+package gitlab
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	gitlab "github.com/xanzy/go-gitlab"
+)
+
+func resourceGitlabServicePipelinesEmail() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceGitlabServicePipelinesEmailCreate,
+		Read:   resourceGitlabServicePipelinesEmailRead,
+		Update: resourceGitlabServicePipelinesEmailUpdate,
+		Delete: resourceGitlabServicePipelinesEmailDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceGitlabServicePipelinesEmailImportState,
+		},
+
+		Schema: map[string]*schema.Schema{
```

Comment:
https://docs.gitlab.com/ee/api/services.html#pipeline-emails

Why omit the `add_pusher` and `pipeline_events` attributes?

Reply:
It does say it in the doc, bu I suspect these fields come from an older version, and are now mapped to new settings. Where trying to create a a pipeline service via the API, these are never returned.
Also, I only mapped what was avaible in the UI.